concepticondata/conceptlists/Hale-1973-1798.tsv

```diff
@@ -1041,7 +1041,7 @@ Hale-1973-1798-1039	Tools_Instruments_Metals-61	carve figure	v	08b.06	228	CARVE
 Hale-1973-1798-1040	Tools_Instruments_Metals-62	bore a hole	v	08b.07	1501	BORE
 Hale-1973-1798-1041	Tools_Instruments_Metals-63	hammer	v	08b.08	3115	HAMMER (SOMETHING)
 Hale-1973-1798-1042	Tools_Instruments_Metals-64	saw	v	08b.09	3543	SAW (SOMETHING)
-Hale-1973-1798-1043	Tools_Instruments_Metals-65	plane	v	08b.10		
+Hale-1973-1798-1043	Tools_Instruments_Metals-65	plane	v	08b.10	2980	AREA
```

Comment:
I would unmap this because "plane" here is a verb, and `2980 AREA` in concepticon is a noun: https://github.com/concepticon/concepticon-data/blob/master/concepticondata/concepticon.tsv#L2976